scripts/v120_dynamic.py

```diff
@@ -0,0 +1,16 @@
+from compas.geometry import Point
+from compas_view2 import app
+
+
+viewer = app.App()
+obj = viewer.add(Point(0, 0, 0))
+
+
+@viewer.animate(interval=1000)
```

Comment:
implemented a decorator for this, what do think? @brgcode

Reply:
cool, but i don't think `animate` is the right name. i imagine this being a very generic function that basically triggers a calculation as defined in the decorated function. for example, to do 1 iteration at every interval of a dynamic relaxation.

`animate` would be something for animating the previously computed stages of the collapse of a masonry vault. (which is also a useful decorator to have)

perhaps something like `on_interval` or `on_timeout`, `on_...`?

what do you think?

Reply:
sounds good, can actually be two optional argument there: `@viewer.on(interval=1000)` , `@viewer.on(timeout=1000)`